### PR TITLE
Add Serilog Logging

### PR DESCRIPTION
**Pull request by MichaelPetrinolis**
Integrate serilog with OrchardCore

### CODE REVIEW

File: src/OrchardCore.Cms.Web/appsettings.json [hunk 1]
```diff
@@ -2,12 +2,11 @@
   "Logging": {
     "IncludeScopes": false,
     "LogLevel": {
-      "Default": "Error",
```
Comment (sebastienros) [inline src/OrchardCore.Cms.Web/appsettings.json]: Information is too verbose by default, and we have Console logging enabled, `Warning` would be fine to me.

**Comment by sebastienros** [comment]
I like that you didn't change the default one we are using and are just documenting how to replace it. We could also have it in the `dotnet new` templates as an option. It's already the case for Nlog.

**Comment by MichaelPetrinolis** [comment]
what about modifying parameter --nlog to --log (-lg) that accepts parameters none,nlog,serilog with default nlog ?

**Comment by sebastienros** [comment]
Did you just create another PR for the same thing? The one I just merged?

**Comment by MichaelPetrinolis** [comment]
Sorry, I made this PR and due to the comment regarding the template parameter I opened a new one which contained this... I close it, it is already merged.... 😊

**Pull request closed without merging**